client/src/apps/Users/Users.module.css

```diff
@@ -0,0 +1,43 @@
+.container {
+  padding: 64px 30px;
+}
+
+.error {
+  display: block;
+  padding: 4px 0;
+  height: 20px;
+  color: red;
+}
+
+.tableWrapper {
+  overflow-x: scroll;
+  width: 100%;
+  max-height: 200px;
+  margin-bottom: 32px;
+}
+
+table {
```

Comment:
W ten sposób w stylach komponentu zmieniasz globalne style dla tagów tabeli. Jeśli rzeczywiście chciałeś to zrobić to nie powinieneś robić tego tutaj. Zamiast tego:
1. W pliku `src/styles.css`
2. Lub jeszcze lepiej może zrobić reużywalny komponent tabeli.

W przeciwnym wypadku w tym miejscu trzymałbym się klas, które potem przy budowaniu będą miały scope modułu.